doc/user-guide.md

```diff
@@ -250,6 +250,101 @@ return reconcile.Result{RequeueAfter: time.Second*5}, nil
 
 **Note:** Returning `Result` with `RequeueAfter` set is how you can periodically reconcile a CR.
 
+#### Reconcile Result Use Cases
```

Comment:
```suggestion
**Example of Reconcile Result Use Cases**
```

Reply:
Hi @camilamacedo86  Thanks for the suggestion.  I had a crazy week last week and wasn't able to look at this but will look at this soon.

Reply:
Hi @camilamacedo86 I just got an email that said the following.  Given what the email says below, should I hold off on trying to update this section of the user guide until docs are fully moved?

We are working to build a documentation website that will replace the in-tree github rendered docs.

The initial bootstrapping is in place, docs are temporarily viewable at operator-sdk.netlify.com. Additionally, all PRs now include a netlify preview build.
MERGED: https://github.com/operator-framework/operator-sdk/pull/2667
MERGED: https://github.com/operator-framework/operator-sdk/pull/2679
existing documentation remains in place under doc/
a copy of the documentation is in website/content/en/docs/
[Important] Please do not merge new docs changes to doc/ unless it is important. If changes must be made, please make them in both doc/ and website/content/en/docs/. We have an additional check in place to help make sure this works.

Reply:
Yes, if we do changes now we need to ensure that the same will be applied in the `website/content/en/docs/` structure as well. 

So, if you would like to move forward with, we need review and be ok with the changes then, after that, do the rebase and ensure that the website structure will have the same content.

Reply:
Hi @camilamacedo86.  Your proposed approach sounds good.  I just did a rebase and will start updating the content as you recommended

Reply:
Hi @camilamacedo86 I have made the revisions to both files following your suggested revisions